libdokan/mounter.go

```diff
@@ -113,24 +119,28 @@ type NoopMounter struct {
 }
 
 // NewNoopMounter creates a mounter that does nothing.
-func NewNoopMounter() NoopMounter {
-	return NoopMounter{c: make(chan struct{}, 1)}
+func NewNoopMounter() *NoopMounter {
```

Comment:
Why did you make this return a pointer?  I don't think there's any reason to for `NoopMounter` to have pointer receivers, since none of its fields are ever modified.